### ISSUE TITLE
elementor beaver widgets

### ISSUE THREAD

**Issue by selul** [Codeinwp/themeisle-companion#51]
@abaicus probably once we have the the multiple headers ready, we should start adding few extra elementor widgets, the most powerful and useful one for different use-cases I think is posts widget, maybe bogdan can also integrate it as a mockup before we start working on it : https://www.dropbox.com/s/4imsvjdhcwjp6lo/Screenshot%202017-07-07%2014.08.49.png?dl=0

example : https://wordpress.org/plugins/void-elementor-post-grid-addon-for-elementor-page-builder/

**Comment by selul** [comment]
moved form old repo.

**Comment by abaicus** [comment]
duplicate of #36
